plugins/slack_app/app/controllers/slack_app_controller.rb

```diff
@@ -0,0 +1,158 @@
+class SlackAppController < ApplicationController
+  skip_before_action :verify_authenticity_token, only: [:command, :interact]
+  skip_around_action :login_user, except: :oauth
+
+  before_action :handle_slack_inputs
+  before_action :verify_environment, except: [:oauth]
```

Comment:
this seems weird ... handle_slack_inputs needs SLACK_VERIFICATION_TOKEN but verify_environment is not called for oauth ... best to make the usage and check be part of the same filter ?